py/torch_tensorrt/dynamo/backend/backends.py

```diff
@@ -20,7 +20,7 @@
 from packaging import version
 
 # Modify import location of utilities based on Torch version
-if version.parse(sanitized_torch_version()) < version.parse("2.1.1"):
+if version.parse(sanitized_torch_version()) <= version.parse("2.1.0.dev20230902"):
```

Comment:
I think it should be `<` and not `<=`, since the September 2 nightly should not enter this code block.

Reply:
I think 2.1.0 will pass this check however since all dev builds are considered < any stable build iirc

Reply:
Add `or` to conditional